base-internal/releases/node-20/20.15.0/README.md

```diff
@@ -0,0 +1,22 @@
+# cypress/base-internal:20.15.0
+A Docker image with all dependencies pre-installed.
+
+NOTE: This image is intended for internal use with https://github.com/cypress-io/cypress. It contains a few differences from the factory, such as:
+
+#### Dependency Additions
+* xauth (to run xvfb inside system-tests)
+* build-essential to install `make` and other linux build packages
+
+#### Env variables
+* Does not contain the `CACHE_FOLDER` and `FACTORY_DEFAULT_NODE_VERSION` env variables to keep unit tests non environment specific
+
+## Example
+
+Sample Dockerfile
+
+```
+FROM cypress/base-internal:20.15.0
+RUN npm install --save-dev cypress
+RUN $(npm bin)/cypress verify
+RUN $(npm bin)/cypress run
+```
```

Comment:
This section is obsolete and no longer works:

---
## Example

Sample Dockerfile

```
FROM cypress/base-internal:20.15.0
RUN npm install --save-dev cypress
RUN $(npm bin)/cypress verify
RUN $(npm bin)/cypress run
```

---

- see PR https://github.com/cypress-io/cypress-docker-images/pull/1125